### DISCUSSION TITLE
Want to Use Powershell..Instead of CMD

### DISCUSSION THREAD

**Discussion by H4ckN3rd** [NvChad/NvChad#3073] [Q&A] [answered]
Hello Everyone, i just started using NVchad, and what to use Powershell instead of CMD, how to do do?

**Comment by siduck** [comment]
@H4ckN3rd  use the `shell` option

**Reply by H4ckN3rd** [reply, marked as answer]
Thanks, it's woking..

vim.o.shell ="<path>"
